sherpa/astro/ui/tests/test_astro_session.py

```diff
@@ -19,15 +20,20 @@
 
 import numpy
 
+import pytest
+
 from sherpa.astro.data import DataPHA
-from sherpa.astro.ui.utils import Session
+from sherpa.astro.ui.utils import Session as AstroSession
+from sherpa.data import Data1D
 from sherpa.models import Const1D
+from sherpa.ui.utils import Session
+from sherpa.utils.err import IdentifierErr
 from sherpa.utils.testing import requires_data, requires_fits
 
 
 # bug #303
 def test_show_bkg_model():
-    session = Session()
```

Comment:
Is there a specific reason this are not parameterized to test both session types? Is that just efficiency of not running unneeded tests? If so, I'm OK with that, I just want to make sure it's intentional.

Reply:
I'll make this as resolved and then it's good for me.